src/main/java/com/example/springboottask/entity/User.java

```diff
@@ -0,0 +1,52 @@
+package com.example.springboottask.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Table(name = "user")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
```

Comment:
Int is ok here, but it is better to use long as an identifier of the entity in enterprise projects

https://stackoverflow.com/questions/17398192/long-versus-integer-for-the-id-field-of-java-entities